components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -56,19 +55,14 @@ export default function () {
     const { teams } = useContext(TeamsContext);
     const team = getCurrentTeam(location, teams);
 
-    const [isLoading, setIsLoading] = useState<boolean>(true);
```

Comment:
❤️ Many thanks for removing a `isLoading` boolean! (which I might have added 🙈)

I agree that maintaining a `isLoading` boolean is generally bad practice, and creates many opportunities for bugs. (And when a loading state is required, it's always better to explicit wait on what still needs to be loaded, instead of trying to code this into a `isLoading` boolean.) 👍